services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -464,13 +464,14 @@ async def retrieve_service_inputs(
         )
         duration = time.time() - started
 
-        get_instrumentation(
-            self.app
-        ).dynamic_sidecar_metrics.input_ports_pull_rate.labels(
-            **get_metrics_labels(scheduler_data)
-        ).observe(
-            get_rate(transferred_bytes, duration)
-        )
+        if rate := get_rate(transferred_bytes, duration) >= 1:
```

Comment:
and same here.
you could at least use the transferred_bytes, which would already be slightly wrong. but rate is even worse.